src/main/java/com/owncloud/android/lib/resources/notifications/RegisterAccountDeviceForNotificationsOperation.java

```diff
@@ -75,8 +75,8 @@ public RegisterAccountDeviceForNotificationsOperation(String pushTokenHash, Stri
 
     @Override
     protected RemoteOperationResult run(OwnCloudClient client) {
-        RemoteOperationResult result = null;
-        int status = -1;
+        RemoteOperationResult result;
+        int status;
```

Comment:
due to this change the initial value is `0` while it has been `-1` before, is this on purpose?

Reply:
Hm. AndroidStudio said that this is overwritten in any case:
status is written in L95 and read in L98, so it should be safe